src/main/java/org/terasology/dynamicCities/construction/Construction.java

```diff
@@ -326,10 +325,10 @@ public int flatten(Rect2i area, int defaultHeight) {
      */
     public ElevationFacet sample(Rect2i area, int height) {
 
-        BaseVector3i minRegionPos = new Vector3i(area.minX(), height - maxMinDeviation, area.minY());
-        BaseVector3i maxRegionPos = new Vector3i(area.maxX(), height + maxMinDeviation, area.maxY());
+        Vector3ic minRegionPos = new org.joml.Vector3i(area.minX(), height - maxMinDeviation, area.minY());
+        Vector3ic maxRegionPos = new org.joml.Vector3i(area.maxX(), height + maxMinDeviation, area.maxY());
```

Comment:
These vectors are only used for creating the region... I think this can be inlined...